ql/src/test/results/clientpositive/llap/materialized_view_no_cbo_rewrite.q.out

```diff
@@ -36,4 +36,8 @@ PREHOOK: query: alter materialized view cmv_mat_view enable rewrite
 PREHOOK: type: ALTER_MATERIALIZED_VIEW_REWRITE
 PREHOOK: Input: default@cmv_mat_view
 PREHOOK: Output: default@cmv_mat_view
-FAILED: Execution Error, return code 40000 from org.apache.hadoop.hive.ql.ddl.DDLTask. org.apache.hadoop.hive.ql.metadata.HiveException: Cannot enable rewriting for materialized view. Statement has unsupported clause: sort by.
+Only query text based automatic rewriting is available for materialized view. Statement has unsupported clause: sort by.
```

Comment:
It seems that with this patch we also enable more rewritting capabilities that we didn't support before. Maybe worth mentioning in the JIRA ticket as well.

Reply:
Actually these capabilities already exists if rewrite is not disabled at creation time. However Alter doesn't allow these. This patch tries to improve Alter too.